cognite/client/_api/data_modeling/__init__.py

```diff
@@ -13,4 +16,7 @@
 class DataModelingAPI(APIClient):
     def __init__(self, config: ClientConfig, api_version: Optional[str], cognite_client: CogniteClient) -> None:
         super().__init__(config, api_version, cognite_client)
+        self.containers = ContainersAPI(config, api_version, cognite_client)
+        self.models = DataModelsAPI(config, api_version, cognite_client)
```

Comment:
I saw Henrik's comment about the double `data_` prefix, but I think I still prefer mirroring the naming in the API.
```suggestion
        self.data_models = DataModelsAPI(config, api_version, cognite_client)
```